### DISCUSSION TITLE
Branchless doesn't seem to play nice with Intellij's “Resolve conflicts” dialog

### DISCUSSION THREAD

**Discussion by LemmingAvalanche** [arxanas/git-branchless#1079] [General]
I am not making an issue since this is not a problem for me. Maybe move this to the issue tracker if someone thinks it belongs there.

I couldn't pick “yours” or “theirs” because Intellij got confused:

![image](https://github.com/arxanas/git-branchless/assets/2229597/dcaa61a3-5008-4745-9d6b-1566bfb9739e)

```
Error saving merged data: git: 'branchless' is not a git command. See 'git --help'.
```

**Comment by claytonrcarter** [comment]
My wild guess is that this related to hooks and paths. eg when you try to save your merge, git will run hooks, but if Intellij is running w/o the `git-branchless` binary in it's `$PATH`, then whatever `git` it's using won't be able to resolve `git branchless` to a command because it won't be able to find the `git-branchless` binary. Again, just a wild guess, but you might try any of these:

- in each of the applicable `.git/hooks` scripts, replace `git branchless` with `/path/to/git-branchless`
- can the `$PATH` be configured in Intellij?
